docs/upgrading.md

```diff
@@ -0,0 +1,111 @@
+# Upgrading Clear Containers installations
+
+This document discusses the available options for upgrading a system running Intel® Clear Containers.
+
+## Terminology
+
+There are currently two implementations of Clear Containers:
+
+- Clear Containers 2.x (or just 2.x) refers to the previous implementation
+  of Clear Containers which uses a runtime called `cc-oci-runtime` and
+  whose code is available at: https://github.com/01org/cc-oci-runtime
```

Comment:
Let's re-work this to avoid the use of "whose":
called `cc-oci-runtime`. 2.x code is available at: https://github.com/01org/cc-oci-runtime